exporter/awsemfexporter/README.md

```diff
@@ -32,12 +32,13 @@ The following exporter configuration parameters are supported.
+| `detailed_metrics`                           | Retain detailed datapoint values in exported metrics (e.g instead of exporting a quantile as a statistical value, preserve the quantile's population)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `false`                                                                                        |
+| `emf_version`                       | Send metrics to CloudWatchLogs with Embedded Metric Format in selected version [(e.g version 1 with _aws)](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html#CloudWatch_Embedded_Metric_Format_Specification_structure), version 0 without _aws)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `1`                                                                                        | 
```

Comment:
the critical thing to communicate to our customers is what the default is (i.e. 1) and that it is optional.  It is also redundant to call it `emf_version` as we are in the emf exporter.  Maybe output_version? I don't care as much about the naming as the lack of documentation on how this should be used (i.e., don't use it, the default is fine :))

Reply:
I disagree `output_version` would make confusing with the `output_destination`. Would ` output_version=2` means we are exporting `emf version v2` to `cloudwatch` or we are exporting  `emf` to `cloudwatch version 2`?

Agree with you on this
```
the critical thing to communicate to our customers is what the default is (i.e. 1) and that it is optional
```

That's why default exists right? For optional configuration instead of force customers to have expertise in it.

Reply:
Change to `version` for less redundant